qiskit_nature/second_q/mappers/interleaved_qubit_mapper.py

```diff
@@ -38,6 +38,17 @@ class InterleavedQubitMapper(FermionicMapper):
        which will indeed produce qubit operators for you. You will just not be able to interpret the
        order of the qubits in the same way.
 
+    .. warning::
+
+        The builtin two-qubit reduction of the :class:`.ParityMapper` will also not provide correct
+        results when combined with this mapper. Again, this is not asserted so be aware of this
+        pitfall.
+
+    .. note::
+
+        Given the above, you may instead want to use a :class:`.TaperedQubitMapper` to remove
```

Comment:
In the output html we have two warnings ahead of this note. `Given the above` seems ambiguous in this context to me - the BKSF warning is basically irrelevant in this regard since BKSF maps to qubits based on interactions, not sites, thus the qubit operator was always different in that regard. Does it in fact make any difference now - I guess it depends on how it builds up the interactions as to whether the re-ordering  makes any difference in the outcome. As it was before, acting on the qubit operator it would have switched things, maybe it still alters the outcome in some way.

![image](https://github.com/Qiskit/qiskit-nature/assets/40241007/765ff35f-4f4e-49d3-9ab1-3f998d54522f)

Maybe just adding a second paragraph to the second warning that suggests if they want reduction to use the TaperedQubitMapper instead. And of course, whether this is worth noting too - where the number of qubits reduced is based on the symmetries it finds rather than always being two based on knowledge of the parity mapper block-mode behavior when its two-qubit reduction is chosen.